RuntimeUnityEditor.UMM/Info.json

```
@@ -0,0 +1,9 @@
+{
+  "Id": "RuntimeUnityEditor",
+  "DisplayName": "Runtime Unity Editor",
+  "Author": "Vanguard",
```

This should be fine, right.
```suggestion
  "Author": "Vanguard, ManlyMarco",
```